lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -19,9 +19,12 @@ Page.refresh = (options = {}, callback) ->
   if options.response
     onlyKeys   = options.onlyKeys   || []
     exceptKeys = options.exceptKeys || []
-    Turbolinks.loadPage null, options.response, true, callback, onlyKeys, exceptKeys
+    Turbolinks.loadPage null, options.response, true, true, callback, onlyKeys, exceptKeys
```

Comment:
I'll look into updating it into a options hash, it is indeed gross. It is not necessarily ignored, but there are other factors that determine whether the window state is updated, and those are never true for XHR-based refreshes.